addons/cargo/CfgVehicles.hpp

```diff
@@ -63,6 +63,7 @@ class CfgVehicles {
         GVAR(space) = 4;
         GVAR(hasCargo) = 1;
     };
+    class Tank_F: Tank {};
```

Comment:
Shouldn't `Tank: LandVehicle` be `Tank_F: LancVehicle`?

Reply:
`Tank_F: Tank` is correct